doc/user/standalone-distribution.md

```diff
@@ -56,7 +56,8 @@ script:
 matrix:
   include:
   - rvm: 2.5.1
-  - rvm: system
+  - name: truffleruby
```

Comment:
I'm a bit hesitating, but I think `TruffleRuby` (with capitalization) would look slightly nicer when looking at https://travis-ci.org/thbar/kiba/builds/424860029
Could you change it? :)